tests/python/relax/test_tvmscript_printer_relax.py

```diff
@@ -47,8 +47,10 @@ def main(a: R.Tensor((10, 10))) -> R.Tensor((10, 10)):
 
 
 def test_extern_func():
+    # note: this function will be treated as private unless a global symbol is added
```

Comment:
This seems annoying but I'm not sure there's anything else we can do for the case of a function outside of an IRModule

Reply:
How about we treat function outside as public always?

Reply:
I'll see if I can detect when a function is not inside an IRModule

Reply:
likely we don;t need such detection? since @R.function itself implies a public function by default i assume and we would need explicit `private=True` flag

Reply:
We don't want to put a global symbol on inner functions (I assume), so that's why the current behavior is to make sure that the function is inside an IRModule.

I think I can detect this case in `entry.py`, however. It might be a little hacky, though.